test/errors/src/index.md

```diff
@@ -19,6 +19,20 @@ Modules = [NonExistantModule]
 NonExistantModule
 ```
 
+```@docs
+# comment in a @docs block
+```
+
+Linking to problematic items: [`ccall`](@ref), [`for`](@ref)
```

Comment:
The (undocumented) syntax for keyword refs is `` [`:ccall`](@ref) ``, though would be better to get it to just handle it properly I suppose.